src/lib/config.ts

```diff
@@ -525,16 +536,20 @@ export class default_config {
     }
 
     /**
-     * Whether to use the keytranslatemap in various maps.
+     * Whether to use the keytranslatemap
      */
-    keytranslatemodes: { [key: string]: "true" | "false" } = {
-        nmaps: "true",
-        imaps: "false",
-        inputmaps: "false",
-        ignoremaps: "false",
-        exmaps: "false",
-        hintmaps: "false",
-    }
+    usekeytranslatemap: "true" | "false" = "false"
```

Comment:
1. I'd like to hear feedback from users too.  
2. After we reach agreement on changes to config itself, we'll add config updater.  
3. Agree on usekeytranslatemap = true by default. But the switch itself may be useful because someone probably wants to disable their map but not lose it?  
4. alwaysqwerty  and usekeytranslatemap are not commands, and i'm not sure there are better names for them. But i18n namespace is a good idea. Or it can be `input` or `keyboard`, i don't mind.